traversal/planner/PlannerEdge.java

```diff
@@ -148,14 +137,9 @@ public String toString() {
     public static abstract class Directional<VERTEX_DIR_FROM extends PlannerVertex<?>, VERTEX_DIR_TO extends PlannerVertex<?>>
             extends TraversalEdge<VERTEX_DIR_FROM, VERTEX_DIR_TO> {
 
-        MPVariable varIsSelected;
-        MPVariable[] varOrderAssignment;
-        private MPVariable varOrderNumber;
-        private int varIsSelected_init;
-        private int varOrderNumber_init;
-        private int[] varOrderAssignment_init;
-        private int varIsSelected_result;
-        private int varOrderNumber_result;
```

Comment:
we move all the `_result` caches and `_init` values into our own optimiser `Variable` classes